testnet/stacks-node/src/config.rs

```diff
@@ -1446,6 +1446,10 @@ pub struct BurnchainConfig {
     /// fault injection to simulate a slow burnchain peer.
     /// Delay burnchain block downloads by the given number of millseconds
     pub fault_injection_burnchain_block_delay: u64,
+    /// The maximum number of unspent UTXOs to request from the bitcoin node.
+    /// This value is passed as the `maximumCount` query option to the
+    /// `listunspent` RPC call.
+    pub max_unspent_utxos: Option<u64>,
```

Comment:
This should probably go in `[miner]`

Reply:
Yeah, I was between the two, since it is only used by miners, but it is related to the burnchain.

Reply:
All of the other miner-specific burnchain options are also in `BurnchainConfig`, e.g. `rbf_fee_increment`, `max_rbf`, etc.